data/assets/scene/digitaluniverse/galaxy_groups.asset

```diff
@@ -33,11 +34,8 @@ local Object = {
   },
   GUI = {
     Name = "Galaxy Group Labels",
-    Path = "/Universe/Galaxies",
-    Description = [[Census: 62 galaxy group labels. DU Version 1.2. <br> The Galaxy
-    Groups data are a set of labels that mark the nearby galaxy groups. The Milky Way
-    is in the Local Group, and we are surrounded by many other groups.(Description
-    from URL) <br><br> Data Reference: Brian Abbott (AMNH)]]
+    Path = "/Universe/Local Universe",
```

Comment:
Might this change of the path be confusing to users? (it was to me)

Reply:
Although on second thought, maybe it does make more sense to split the galaxies up in Deep Sky VS Local.  Hard for me to have an opinion here